src/go/rpk/pkg/cli/cmd/cluster/config/status.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+)
+
+func newStatusCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "status",
+		Short: "Get configuration status of redpanda nodes",
+		Long: `Get configuration status of redpanda nodes
+
+For each node, indicate whether a restart is required for settings to
```

Comment:
This help text looks a bit narrow -- if I copy locally into vim and `gwap`, a few words move around.

Reply:
This is 70 chars wide, which I think is readable.  When I point vim at this, it tries to indent everything, and consequently the lines are shorter than 80 chars after I unindent the result.